tools/README.md

```diff
@@ -167,3 +167,64 @@ The output chunked graph metadata will go as follows (assuming the current direc
     "edge_data": {}
 }
 ```
+
+## Partition Config Conversion
+
+`convert_partition_conf.py` is a tool help you convert old partition config to new version.
```

Comment:
Be more explicit about what you try to convert. Readers won't know what do you mean by old version or new version. I will also change the title to `Rewrite edge type to canonical edge type for partitioned graph data`.

Reply:
Got it and changed. At first we use etype2canonicaletype as the name of the file, but consider that there maybe extensions in the future, we changed it to a more general name. However, because it's a temporary tool and dedicated to etype->canonical etype feature, use a more specific name is better.